index.js

```diff
@@ -147,7 +148,7 @@ function search(tree, phrases, handler, options) {
             if (
                 !node ||
                 node.type !== T_WORD ||
-                normalize(expression[index], apos) !== normalize(node, apos)
+                normalize(expression[index], apos, dashes) !== normalize(node, apos, dashes)
```

Comment:
When I change the variables `apos` and `dashes` to hardcoded `false` and `true` respectively, everything seems to work properly all the way into rorybot. So I think the issue has something to do with the way these variables are getting passed back to the normalize() function.